package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "passport-vkontakte",
-  "version": "0.3.2",
+  "version": "0.3.5",
```

Comment:
This isn't necessary, I will bump the version myself, probably to a `0.4.0` because it enables new capabilities. Could you revert it, please?